.github/workflows/pytest.yml

```diff
@@ -20,7 +20,7 @@ jobs:
 
     steps:
       - name: "Checkout the repository"
-        uses: actions/checkout@v4
+        uses: actions/checkout@v3
```

Comment:
this change needs to be moved together with the commit that adds the github workflow, and mention that v4 uses a too much newer nodejs that does not run on EL7